cmd/minikube/cmd/delete.go

```diff
@@ -601,49 +602,99 @@ func killMountProcess() error {
 	return nil
 }
 
+// killProcess takes a path to look for a pidfile(space-separated),
+// it reads the file and converts it to a bunch of pid ints,
+// then it tries to kill each one of them.
+// If no errors were ecnountered, it cleans the pidfile
 func killProcess(path string) error {
 	pidPath := filepath.Join(path, constants.MountProcessFileName)
 	if _, err := os.Stat(pidPath); os.IsNotExist(err) {
 		return nil
 	}
-
 	klog.Infof("Found %s ...", pidPath)
-	out, err := os.ReadFile(pidPath)
+
+	ppp, err := GetPids(pidPath)
 	if err != nil {
-		return errors.Wrap(err, "ReadFile")
+		return err
 	}
-	klog.Infof("pidfile contents: %s", out)
-	pid, err := strconv.Atoi(string(out))
-	if err != nil {
-		return errors.Wrap(err, "error parsing pid")
+
+	// we're giving a change to each pid, to be killed
```

Comment:
I'm not sure what this comment means, could you explain it?

Reply:
We're not stopping on the first error found while trying to kill mount processes.
Instead we're trying to send the signal to all mount processes, capturing eventual errors; then doing error handling logic

Reply:
oh.. it was supposed to be `chance`, not `change`. sorry

I modified it.. now it should be more readable